README.rst

```diff
@@ -1003,14 +1003,43 @@ Admin Multitenancy mixins
     <https://github.com/openwisp/openwisp-firmware-upgrader/search?q=multitenant_parent>`_
     for a real world example.
 
-* **MultitenantOrgFilter**: admin filter that shows only organizations
-  the current user can manage in its available choices.
+* **MultitenantOrgFilter**: an autocomplete admin filter that shows only organizations
+  the current user can manage in its available choices. The following example
+  adds the autocomplete organization filter in ``BookAdmin``:
+
+.. code-block:: python
+
+    from django.contrib import admin
+    from openwisp_users.multitenancy import MultitenantOrgFilter
+
+    class BookAdmin(admin.ModelAdmin):
+        list_filter = [
+            MultitenantOrgFilter,
+        ]
+        # other attributes
+
 
 * **MultitenantRelatedOrgFilter**: similar ``MultitenantOrgFilter`` but
   shows only objects which have a relation with one of the organizations
-  the current user can manage, this shall be used when the model does not
-  have its own organization field but relies on a parent model which
-  has the organization field.
+  the current user can manage, this shall be used for creating filters
+  for related objects. In the following example, an autocomplete filter is
+  created for ``shelf`` field of ``BookAdmin``.
+
+.. code-block:: python
+
+    from django.contrib import admin
+    from openwisp_users.multitenancy import MultitenantRelatedOrgFilter
+
+    class ShelfFilter(MultitenantRelatedOrgFilter):
+        field_name = 'shelf'
+        parameter_name = 'shelf'
+        title = _('Shelf')
+
+    class BookAdmin(admin.ModelAdmin):
+        list_filter = [
+            ShelfFilter,
+        ]
+        # other attributes
```

Comment:
Why don't we add examples from other OpenWISP modules and link them? I think it will be a lot easier to understand, we should also provide basic info about the relationships between these models. I believe this example is not easy to understand for a newcomer.